test/TestApps/ToastNotificationsDemoApp/main.cpp

```diff
@@ -175,7 +148,8 @@ int main()
         const PACKAGE_VERSION minVersion{};
         RETURN_IF_FAILED(MddBootstrapInitialize(c_Version_MajorMinor, nullptr, minVersion));
 
-        SetDisplayNameAndIcon();
+        // Not mandatory, but it's highly recommended to specify AppUserModelId
+        THROW_IF_FAILED(SetCurrentProcessExplicitAppUserModelID(L"TestAppId"));
```

Comment:
@vaheeshta This is needed if we want apps to maintain their toast states even after a process name update later in the future. For example, if unpackaged app foo.exe has a name change to bar.exe in the future, this ensures that the apps toasts remain in the actioncentre. We should document.

Reply:
It also is needed if apps change their path from c:\foo\name.exe to d:\bar\name.exe.

Reply:
Got it. I'll document this once I have the unpackaged sample code.